### PR TITLE
chore: update aws qldb quickstart

### PR DESCRIPTION
**Pull request by aliandersonsn**
Search Nurture recommended updates to quickstart description

# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ ] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/aws/aws-qldb/config.yml [hunk 1]
```diff
@@ -3,22 +3,29 @@ name: aws-qldb
...
-  Create a cryptographically verifiable ledger database that's managed by one
-  group.
+  Amazon Quantum Ledger Database (AWS QLDB) is a fully managed ledger database that provides a transparent, immutable, and cryptographically verifiable transaction log. Proactively instrument AWS QLDB with the New Relic infrastructure monitoring agent to monitor the stability and reliability of AWS QLDB.
 
 
-  ### Get started!
+  ### AWS QLDB quickstart highlights
+  
+  The New Relic AWS QLDB quickstart includes the following features
+  Dashboards- Proactively monitor metrics like ledger samples, provider readIOs latency, ledgers, overview of ledger provider, etc.
```
Comment (jpvajda) [inline quickstarts/aws/aws-qldb/config.yml:12]: This seems like a typo, did you want a bullet here?

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: quickstarts/aws/aws-qldb/config.yml (hunk 1 above)
Comment (aliandersonsn) [reply quickstarts/aws/aws-qldb/config.yml:12]: So I'd like this to be a colon, but I'm not sure if that's possible with the code.

Comment (jpvajda) [reply quickstarts/aws/aws-qldb/config.yml:12]: @aliandersonsn oh yes, I think you can use `:` in the YAML as it's treated like markdown. We can test it out and see if it passes the checks.

Comment (jpvajda) [inline quickstarts/aws/aws-qldb/config.yml:12]:
```suggestion
  Dashboards: Proactively monitor metrics like ledger samples, provider readIOs latency, ledgers, overview of ledger provider, etc.
```

**Review by jpvajda — CHANGES_REQUESTED** [review]
I made a suggest edit

**Review by jpvajda — APPROVED** [review]

**Pull request merged**